Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java

```diff
@@ -93,7 +89,7 @@ public void initialize(final MaxAdapterInitializationParameters parameters, fina
 
             status = InitializationStatus.INITIALIZING;
 
-            Plugin.addWrapperInfo( Plugin.WrapperFramework.max, getAdapterVersion() );
+            VungleAds.setIntegrationName(WrapperFramework.max, getAdapterVersion() );
```

Comment:
You're missing a space after the ( that MAX wants in their syntax.

Is Android syntax the same as iOS syntax, where they want `VungleAds.setIntegrationName( WrapperFramework.max, getAdapterVersion() );`?

Reply:
Never mind. Looking at the code below, it looks like MAX is not as strict about the styling as they are in iOS.